tests/test_frontend.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import unittest
+from selenium import webdriver
+from flask import request
+
+class TestFrontEnd(unittest.TestCase):
+
+    def test_web_app_running(self):
```

Comment:
This test is checking that the app is running and that you can find an HTML element on the page.
I think this is also tested by the next test (add to contacts) no? If so, then you can remove this first test as it does not add value to your test suite.

Reply:
I left it because it tells me that the page fired up. If the specific test is failing but the test_web_app_running is passing then I know that it's the specific feature and not an issue with webdriver/setup etc.